maidr/custom_renderers/shiny_renderer.py

```diff
@@ -0,0 +1,63 @@
+from matplotlib.axes import Axes
+from matplotlib.figure import Figure
+import maidr
+from shiny import ui
+from htmltools import tags
+from typing import Any, Optional, Callable, TypeVar, Union, List
+
```

Comment:
Are `matplotlib` and `htmltools` used after you import?

Reply:
Yes both imports are not used after they are imported. Remnants from the older version of the `shiny_renderer` and they have been removed in the latest commit.